src/frontend/overlay/src/components/molecules/Clock.jsx

```diff
@@ -32,8 +33,8 @@ export const ContestClock = ({
     }, [contestInfo, contestCountdownMode]);
 
     const getStatus = useCallback(() => {
-        if (globalTimeMode === true) {
-            return DateTime.now().setZone(new SystemZone()).toFormat(quietMode ? "HH:mm" : "HH:mm:ss");
+        if (globalTimeMode === true || timeZone !== "") {
```

Comment:
I don't think it's a good idea to set the clock to global if the timezone was specified. This might be confusing to the users